### DISCUSSION TITLE
Hidden for different resolutions

### DISCUSSION THREAD

**Discussion by Frankusky** [tailwindlabs/tailwindcss#13307] [Ideas]
I need to hide an element only for tablets and show it again on mobile and desktop, looks like the **hidden** class doesnt have the breakpoints prefix available, also using visible or invisible class wont help as it leaves the empty space. The following classes wont do the trick:

hidden sm:block lg:block

**Comment by andymccray** [comment]
The `hidden` class should definitely work with the breakpoint prefixes. For this situation I’d go:

```html
<div class="md:hidden lg:block">…</div>
```

… assuming `md` is your tablet breakpoint and `lg` is your desktop breakpoint. The element will be `block` by default (assuming it’s a block-level element) on mobile, unless you’ve explicitly set it otherwise.

**Comment by Frankusky** [comment]
Oh nvm, after debugging I found that there was a display:flex in a css module that was overwriting the md:hidden, thanks for your help
